src/flush-css-chunks-webpack-plugin.js

```diff
@@ -1,8 +1,17 @@
 const { ConcatSource } = require('webpack-sources');
 const { matchObject } = require('webpack/lib/ModuleFilenameHelpers');
 
+const pluginName = 'flush-css-chunks';
+
 function isInitial(chunk) {
-  return chunk.isInitial() || chunk.parents.length === 0;
+  let parentCount = 0;
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const chunkGroup of chunk.groupsIterable) {
```

Comment:
You can avoid the eslint disable comment:

```js
Array.from(chunk.groupsIterable).forEach(function(chuckGourp) {
  parentCount += chunkGroup.getNumberOfParents();
})
```

Reply:
Yes but i prefer performance !